src/pint/solar_system_ephemerides.py

```diff
@@ -109,12 +109,18 @@ def _load_kernel_local(ephem, path):
         custom_path = os.path.join(path, ephem_bsp)
     else:
         custom_path = path
-    search_list = [custom_path, datapath(ephem_bsp)]
+    search_list = [custom_path]
+    try:
+        search_list.append(datapath(ephem_bsp))
+    except FileNotFoundError:
+        # If not found in datapath, just continue. Error will be raised later if also not in "path"
+        pass
     for p in search_list:
         if os.path.exists(p):
             # .set() can accept a path to an ephemeris
             coor.solar_system_ephemeris.set(ephem)
-
+    # Question(paulr): How does this work? It looks like this function should always get to this exception!
```

Comment:
@demorest Can you look at this? I don't understand how it ever worked. It looks like it goes through the search_list and calls `.set(ephem)` for any path that exists, but then it should fall to the raise. How did it not always raise an exception?

Reply:
yeah that looks pretty busted to me.  I think you're right that it will always raise an exception.  looks like that function is only called from within a try/except that ignores the exception.. probably should understand this and clean it up.

Reply:
OK, thanks! I'll give a shot at cleaning it up and add that to this PR

Reply:
OK, I pushed some fixes. I don't think load_kernel_local ever worked, it was just hidden by try/except blocks.